spec/fixtures/iso-sample-identifiers-v2.txt

```diff
@@ -3216,7 +3216,8 @@ ISO 1927-7:2012
 ISO 17693:2004
 ISO 19860:2005
 ISO 11003-2:2001
-ISO 10438-1:2003 EN 80601-2-60
+ISO 10438-1:2003
+IEC 80601-2-60
```

Comment:
Let's not parse IEC identifiers here. IEC identifiers can go into a separate gem, the numbers are different.

Reply:
Separated non-ISO identifiers into a different file now.